### ISSUE TITLE
clipping not working

### ISSUE THREAD

**Issue by mayeulk** [kartoza/docker-osm#66]
https://github.com/kartoza/docker-osm/blob/b1a780088e0a660d3a32fb3f12f4454c14366cde/Makefile#L94

**Comment by mayeulk** [comment]
Several issues (at least with my settings):
- /settings folder does not exist in _db container
- _db container in my settings get named *_db_1  (while I start with no _db container). So this container gets not found
- no shp2pgsql executable in in *_db container

**Comment by mayeulk** [comment]
How I (temporarily) fixed this: 

working_folder=~/bin/dockerosm_1
# specific dockerosm version (sha) to download
dockerosm_sha=b1a780088e0a660d3a32fb3f12f4454c14366cde
# readable name (for instance, date) to give to this version (and the folder where it will be unzipped):
dockerosm_version=20180622
# stop / remove all of Docker containers:

docker stop $(docker ps -a -q)
docker rm $(docker ps -a -q)

mkdir -p $working_folder
cd $working_folder
# Version : 22 June 2018
wget -O develop.zip https://github.com/kartoza/docker-osm/archive/$dockerosm_sha.zip
unzip ./develop.zip
mv docker-osm-$dockerosm_sha docker-osm-$dockerosm_version
cd $working_folder/docker-osm-$dockerosm_version/settings
wget  -c -O country.pbf https://download.geofabrik.de/africa/south-africa-latest.osm.pbf
# get clip shapefile into $local_folder (FIXME: not shown), then move it:
cp $local_folder/south_africa/clip.* $working_folder/docker-osm-$dockerosm_version/settings/clip

# In Makefile, replace:
# @docker exec -t -i $(PROJECT_ID)_db /usr/bin/shp2pgsql -c -I -D -s 4326 /home/settings/clip/clip.shp | docker exec -i $(PROJECT_ID)_db su - postgres -c "psql gis" 
# by:
# @docker exec -t -i docker-osm-20180622_imposm_1 /usr/bin/shp2pgsql -c -I -D -s 4326 /home/settings/clip/clip.shp | docker exec -i docker-osm-20180622_db_1 su - postgres -c "psql gis"

docker-compose build
docker-compose up

**Comment by mayeulk** [comment]
Also, to have the shp2pgsql command in imposm container, I do this (as root):
```
apt-get update
apt-get install postgis

```
Possible not the most disk-savy solution, as "After this operation, 761 MB of additional disk space will be used." But then, clipping works, after running the following and waiting two minutes:
```
cd $working_folder/docker-osm-$dockerosm_version
make import_clip
```

**Comment by mayeulk** [comment]
It should also be note that, after the above, make import_clip returns an error (below), but clipping works:
```
~/bin/dockerosm_1/docker-osm-20180622>make import_clip

------------------------------------------------------------------
Importing clip shapefile
------------------------------------------------------------------
ERROR:  syntax error at or near "Shapefile"
LINE 1: Shapefile type: Polygon
        ^
SET
BEGIN
CREATE TABLE
ALTER TABLE
                  addgeometrycolumn                   
------------------------------------------------------
 public.clip.geom SRID:4326 TYPE:MULTIPOLYGON DIMS:2 
(1 row)

COPY 1
CREATE INDEX
COMMIT
ANALYZE

```
Also, connecting to the database shows that a geometry was indeed created in table "clip" (db: gis; schema: public)

**Comment by Gustry** [comment]
This should be fixed with #72
